src/Atmos/Model/AtmosModel.jl

```diff
@@ -173,10 +178,33 @@ struct AtmosModel{
     tracers::TR
     "Large-scale forcing (Forcing information from GCMs, reanalyses, or observations)"
     lsforcing::LF
+    "Compressibility switch"
+    compressibility::C
```

Comment:
@tapios If we change `Incompressible` to `SingleStack`, then what should `compressibility` be? My impression was that everything inside `AtmosModel` should be physics-centric, and agnostic to the configuration (GCM/LES/SingleStack). We're okay with making `config::SingleStack` a property of `AtmosModel`?

Reply:
Spoke with @ilopezgp about this. Thanks!